src/components/SimplePagination/_simple-pagination.scss

```diff
@@ -0,0 +1,28 @@
+@import '~carbon-components/scss/globals/scss/vars';
+
+div.simple-pagination-container {
+  display: flex;
+  height: 3rem;
+  justify-content: flex-end;
+  width: 100%;
+  border: 1px solid $ui-03;
+  background-color: $ui-background;
+  align-items: center;
+
+  span.simple-pagination-page-label {
+    padding-right: 1rem;
```

Comment:
should use the Carbon spacing token here

Reply:
Fixed